### ISSUE TITLE
1.15.2: Disable DiscordSRV Join/Quit messages on server switch

### ISSUE THREAD

**Issue by LeehamElectronics** [kangarko/ChatControl-Pro#1837]
Currently running a few bukkit servers connected with bungee. I use the latest version of ChatControlPro and BungeeControl.

At the moment everything is set up perfectly, but since all 3 servers are connected to the same Discord channel there is a lot of spam from players switching from server to server. Is it possible to disable join and quit messages being sent to Discord if the player is switching servers? I know I can just disable join and quit messages in the DiscordSRV configs of all the bukkit servers except for the main server, but that would create another issue because when players leave the game in one of the other servers no quit message will be sent to Discord at all.
So basically my question is, how can I disable sending join / quit messages to Discord is the player is switching servers?
Even better... Would it be possible to instead send a similar message to what BungeeControl sends? For example, send Discord a message saying "(player) has switched from (server) to (server)." This would half the amount of spam which you can see in the image...
![image](https://user-images.githubusercontent.com/51737378/90087947-40fb5200-dd61-11ea-91bd-b78b2266119e.png)

Thanks!

**Comment by AngellitoX** [comment]
Hello, nice to see you again.

I don't think there's any way to do that currently from what I know (I personally and extensively use DiscordSRV for 5 of my servers) Also this looks like it's a DiscordSRV question.

I have one curious question though, why would you have 3 servers connected to the same Discord channel? It sounds a bit crowded for that channel.

**Comment by LeehamElectronics** [comment]
Hey thx,
I have all 3 in the same Discord channel so that way my players can talk to there friends in Discord easily no matter which server they are in. This is because I need it to be simple for players and the only reason I have 3 servers (SkyBlock, Main, and Survival) Is to keep inventories and economies separate so I wanted to keep all things "Chat related" together for simplicity. 
Okay ill ask DiscordSRV on there Discord server for help and I will post here the answer.

Thanks

**Comment by LeehamElectronics** [comment]
I contacted DiscordSRV and this is the reply I got, should be fixed by the "end of summer"
![image](https://user-images.githubusercontent.com/51737378/90098725-39957200-dd7c-11ea-9042-470c9139afb0.png)

**Comment by AngellitoX** [comment]
Happy to know that!

I'm closing this ticket for now. Feel free to reopen it in case there's anything related to it eventually.

Thank you for your time!.
